replay/models/nmslib_hnsw.py

```diff
@@ -273,11 +274,11 @@ def build_index(iterator):
                 if index_type == "sparse":
                     item_vectors.select(
                         "similarity", "item_idx_one", "item_idx_two"
-                    ).mapInPandas(build_index, "_success int").show()
+                    ).mapInPandas(build_index, "_success int").collect()
```

Comment:
pls, add here a comment about what happens inside build_index function because the place looks unsual.
I mean, collect() assumes that you return something but it is not the case here. Instead you use it to materialize the index and save right from 'build_index' functiom/